examples/models/semantic_segmentation/deeplab_v3/pascal_voc/train.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2022 The KerasCV Authors
```

Comment:
Can we move this to `examples/training/semantic_segmentation/pascal_voc/deeplab_v3.py`?

(Or something generic in that folder like `basic_training.py` if this can support other models)

Reply:
Sounds good. task/dataset/model is better than task/model/dataset